src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.async.txt

```diff
@@ -52,8 +52,7 @@
    50|      1|    // non-async versions of `c()`, `d()`, and `f()` to make it similar to async `i()`.
    51|      1|    fn c(x: u8) -> u8 {
    52|      1|        if x == 8 {
-   53|      1|            1 // This line appears covered, but the 1-character expression span covering the `1`
-                          ^0
+   53|      0|            1 // This line appears covered, but the 1-character expression span covering the `1`
```

Comment:
These Rust coverage results should not have changed.

I believe this is a regression introduced by https://reviews.llvm.org/D85036. 

The change was reverted yesterday (Feb 19): https://reviews.llvm.org/D85036#2575986

As it turns out, I'm very familiar with this patch, and the problems it caused for Rust, because I was asked about unexpected results like these, and I had to prove that the new `llvm-cov` results (after the patch) were actually wrong. They don't represent the actual regions provided in Rust's coverage map.

So they are now fixing their coverage map in Clang instead.

I had hoped there would be more time between the revert and our upgrade to LLVM 12, but hopefully you can get that revert now, as it is important.

Please make sure you update LLVM 12 to include the reverted patch to llvm-cov, and then re-bless the coverage results.

Hopefully all of these changes to test results will no longer be there.